core/embed/trezorhal/stm32u5/bg_copy.c

```diff
@@ -31,7 +32,7 @@ void HAL_DMA_XferCpltCallback(DMA_HandleTypeDef *hdma) {
   }
 }
 
-void GPDMA1_Channel0_IRQHandler(void) {
+__attribute__((optimize("-O0"))) void GPDMA1_Channel0_IRQHandler(void) {
```

Comment:
why O0?

Reply:
It was here just for debugging, I forgot to remove it.. fixed here 294c6d52894e33d6a19dae14f4dd42734faa0c57